README.md

```diff
@@ -170,15 +185,41 @@ modules.exports = [
 ]
 ```
 
-See `examples/middleware/` for some meatier examples.
+See `examples/middleware/` for some meatier examples. Pro tip: debugging is easier if you name each of these layers instead of using anonymous functions.
```

Comment:
TODO, someday: link to naming conventions for middleware. ("attachX", "verb", etc.)